js/main.js

```diff
@@ -0,0 +1,35 @@
+var fieldWidth ;
+var fieldHeight;
+var minesCount;
+var startGameDiv = document.getElementById("start-game");
+var buttonsHolderElement = document.getElementById("buttons-holder");
+var smallFieldButton = document.getElementById("small-field");
+var largeFieldButton = document.getElementById("large-field");
+var buttonsElements = buttonsHolderElement.getElementsByTagName("button");
+var countOfMines = document.getElementById("mines-counter-holder");
+var messageElement = document.getElementById("message");
+var tryAgainButton = document.getElementById("try-again");
+var fieldElement = document.getElementById("field");
+
+for (var i = 0; i < buttonsElements.length; i++) {
+	buttonsElements[i].onclick = function (e) {
+		countOfMines.classList.remove("hidden");
+
+		startGameDiv.classList.add("hidden");
+		if (e.target === smallFieldButton) {
+			fieldWidth = 15;
+			fieldHeight = 10;
+			minesCount = 15;
+		} else {
+			fieldWidth = 25;
+			fieldHeight = 12;
+			minesCount = 50;
+		}
```

Comment:
Such settings better to store at the top of code, for example as two objects like - smallFieldSettings - it will be easier to update these values later if needed.